client/state/importer-nux/actions.js

```diff
@@ -6,7 +6,6 @@ import {
 	IMPORT_IS_SITE_IMPORTABLE_ERROR,
 	IMPORT_IS_SITE_IMPORTABLE_RECEIVE,
 	IMPORTER_NUX_URL_INPUT_SET,
-	IMPORTER_NUX_URL_VALIDATION_SET,
```

Comment:
Remove url validation message from global state since it isn't shared outside component.